migration/v1_to_v2/configuration/exporters/configuration_exporter.rb

```diff
@@ -108,6 +108,70 @@ def config_to_ruby_string(config_name, config_hash)
     ruby_string + "\n#{i})\n\n"
   end
 
+  def migrate_config_fields_translations(object, parent_lang='')
+    translation_fields = [
+      'description_',
+      'display_name_',
+      'guiding_questions_',
+      'help_text_',
+      'name_',
+      'tick_box_label_',
+      'upload_document_help_text_',
+      'tally_',
+    ]
+    translation_array_fields = [
+      'option_strings_text_',
+      'lookup_values_'
+    ]
+
+
+    config_object = Marshal.load(Marshal.dump(object))
+    if object.kind_of?(Array) && parent_lang.empty?
+      return config_object.map { |item|
+        migrate_config_fields_translations(item)
+      }
+    end
+
+    config_object.keys.each do |key|
+      m = key.match(/^(#{(translation_fields).join('|')})([a-z]{2}|[a-z]{2}[_-]{1}[A-Z]{2})/)
```

Comment:
please avoid one|two single characteres variables.
Can you create a separate method that return true or false instead of repeat the regex